packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -64,8 +64,19 @@ export class Meter implements types.Meter {
       );
       return NOOP_MEASURE_METRIC;
     }
-    // @todo: implement this method
-    throw new Error('not implemented yet');
+    const opt: MetricOptions = {
+      absolute: true,
```

Comment:
Suggest to add a comment here: 
`// Measures are defined as absolute by default` being consistent with other two create methods :)